projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/zoom.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:zoom",
+  "text": [
+    {
+      "text": "找找周围的东西，一个在很远很远地方的东西。"
```

```suggestion
      "text": "找找周围的东西，一个在很远很远地方的东西。"
```
在很远的地方能叫周围么（
I spy with my little eyes是一个小游戏，也许这里可以考虑换成别的翻译？比如“找啊找啊找山头~ 找到一个小山头~”

---

这里的话，那个小游戏是找到了，但是不怎么会翻
就按歌谣的形式翻吧